zqd/api/api.go

```diff
@@ -104,6 +104,12 @@ type SpacePostResponse SpacePostRequest
 
 type PacketPostRequest struct {
 	Path string `json:"path"`
+	// SortLimit specifies the limit of logs in posted pcap to sort. Its
```

Comment:
It would be nice to be able to avoid broadening the API for testing support where it can be avoided... did you consider making this a constant in package `packet` and just modifying it for the test?

Reply:
Normally I would be one the making this comment, however I believe @alfred-landrum is of the school of its okay to add hooks specifically for testing. That said, I could totally change package.DefaultSortLimit to be a `var` instead of a `const`- the down side is we would have to make sure this wasn't running in parallel.

Reply:
It's not parallel right now, but hopefully will be, and I agree we should be parallel-proof. Which would mean this would have to be added into `Core` or something which is also intrusive. Well, if we're ok having testing fields in the public API, then I guess let's!

Reply:
I don't think we should expose the ability to override the limit in the REST api, since that's exposed to end users of the service. I think there should be a way to setup the test server so that when it calls IngestFile, it calls it with a non-default.

Reply:
Ok, I'll change this.

Reply:
Alright changed. I put the setting zqd.Core.